pyzscaler/zia/web_dlp.py

```diff
@@ -0,0 +1,82 @@
+import json
+
+import requests
+from box import Box, BoxList
+from restfly.endpoint import APIEndpoint
+
+
+class WebDLP(APIEndpoint):
+    def get_all(self, **kwargs) -> BoxList:
```

Comment:
Are you happy if we rename these methods to align with the rest of the SDK? 

I've taken the approach that Zscaler may add additional objects to some of these API endpoints, which would mean we need to be a bit more verbose about the verbs and what we're doing. So at the moment it appears that the web_dlp endpoint just has policy rules, but if they were to add anything else then just having `zia.web_dlp.get_all() `would be ambiguous.

So while it may seem a bit over the top, having a descriptive verb/noun for the methods ensures we don't need to go back and refactor if the API endpoints are expanded. You should see a _module.attribute.verb_noun_ convention used across the SDK.

If we're returning a list of items then we should expect something like `zia.web_dlp.list_rules()`